tests/test_cpp/polars/TestWindowProcessor.cpp

```diff
@@ -213,6 +213,32 @@ namespace WindowProcessorTests {
         ) << "Expect " << "with window=3, min_periods=2 the edge values should be NAN and the rest should sum the windows";
     }
 
+    TEST(Series, rolling_mean_normal) {
+        EXPECT_PRED2(
+                Series::equal,
+                Series().rolling(5, polars::Mean(), 1),
+                Series()
+        )  << "Expect " << "empty Series returns empty Series";
+
+        EXPECT_PRED2(
+                Series::equal,
+                Series({1,2,3,4,3,2,1}, {1,2,3,4,5,6,7}).rolling(5, polars::Mean(), 1),
+                Series({2, 2.5, 2.6, 2.8, 2.6, 2.5, 2}, {1,2,3,4,5,6,7})
+        )  << "Expect " << "empty Series returns empty Series";
```

Comment:
Think this expectation comment is faulty 😂

Reply:
- [ ] sort this expectation string out